libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -279,10 +279,14 @@ void ExtVotesPacketHandler::onNewPbftVote(const std::shared_ptr<Vote> &vote, con
     if (!(vote->getType() == cert_vote_type && vote->getBlockHash() == rewards_votes_block.first)) {
       // CONCERN ... should we be using a period stored in peer state?
       if (peer.second->pbft_chain_size_ > vote->getPeriod() - 1) {
+        LOG(log_dg_) << " PBFT vote " << vote->getHash() << " not sent to " << peer.first
```

Comment:
I think it is worth to add such logs to ifs above(especially syncing check). For me it looks like more probable to fail there